### PR TITLE
SEO-176062-Typscript--control-issue-fixes

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Have made changes on the redirect links. Please review.
Thanks.

Regards,
Edith.

### CODE REVIEW

File: TypeScript/DatePicker/How-To.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ You can customize the date field to emphasize the particular dates in DatePicker
 
 ## Localize DatePicker with browser specific culture
 
-You can get the browser culture name at page load or document ready state. Based on the culture name, DatePicker can be initiated with that specific culture value by assigning to [locale](https://help.syncfusion.com/api/js/ejdatepicker#members:locale) property. Refer the sample from the link [Browser Specific Culture](https://support.syncfusion.com/kb/article/4702/how-to-change-the-culture-of-the-datepicker-control-based-on-the-browser-language) to create DatePicker with browser specific culture.
+You can get the browser culture name at page load or document ready state. Based on the culture name, DatePicker can be initiated with that specific culture value by assigning to [locale](https://help.syncfusion.com/api/js/ejdatepicker#members:locale) property. Refer the sample from the link [Browser Specific Culture](https://support.syncfusion.com/kb/article/4702/how-to-change-the-culture-of-the-datepicker-control-in-javascript?isInternalRefresh=False) to create DatePicker with browser specific culture.
```
Comment (Yvone-Atieno) [inline TypeScript/DatePicker/How-To.md:17]: Check this KB link

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Fine.

**Pull request merged**